client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -280,6 +307,38 @@ export default () => {
               </Grid>
             </Box>
 
+            {/* Participant RoS Info */}
+            <CheckPermissions
+              permittedRoles={['health_authority', 'employer', 'ministry_of_health']}
+            >
+              {participant.ros && (
+                <>
+                  <Box pt={4} pb={2} pl={4} pr={4}>
+                    <Typography variant='h2'>Return of Service</Typography>
+                  </Box>
+
+                  <Box py={2} px={4}>
```

Comment:
We can probably remove this `Box` and update spacing for the upper Box to `p={4}`.
I really don't like the idea of wrapping a `Grid` with `Box` component. Especially, since you're using `className`. You can specify the horizontal spacing in there instead

Reply:
As I discussed with you, the whole page is developed in a mixed approach, so I created this section in the same manner. I created this [task](https://freshworks.atlassian.net/browse/HCAP-1241) to implement a uniform layout structure.

Reply:
I'm okay with keeping existing structure. Yet I don't see any reason to introduce more Box/Grid issues, especially when there's a workaround